Basics Practice/problem4.py

```diff
@@ -0,0 +1,7 @@
+# The provided code stub reads and integer, n, from STDIN. For all non-negative integers i<n, print i^2.
```

Comment:
This is already present in this repo at https://github.com/Rishit-dagli/Python3-by-practice/tree/master/Training%20Modules/01%20Introduction/1.05%20Loops